docs/getting-started/fundamentals-block-development/file-structure-of-a-block.md

```diff
@@ -0,0 +1,88 @@
+</div>
+
+### `package.json`
+
+[`package.json`](https://docs.npmjs.com/cli/v10/configuring-npm/package-json) is a configuration file for a NodeJs project. In this file you define the NPM dependencies of the block and the scripts used for local work.
+
+### `src` folder
+
+In a standard project you'll place your block files in the `src` folder. By default, the build process with `wp-scripts `will take files from this folder and will generate the bundled files in the `build` folder. 
+
+### `block.json`
+
+This file contains the metadata of the block, and it's used to simplify the definition and registration of the block. 
+
+Among other data it provides properties to define the paths of the files involved in the block's behaviour, output and style. If there's a build process involved, this `block.json` along with the generated files are placed into a destination folder (usually the `build` folder) so the paths provided target to the bundled versions of these files.
+
+The most relevant properties that can be defined in a `block.json` to set the files involved in the block's behaviour, output or style are:
+- The `editorScript` property, usually set with the path of a bundled `index.js` file (output build from `src/index.js`).
+- The `style` property, usually set with the path of a bundled `style-index.css` file (output build from `src/style.scss`).
+- The `editorStyle` property, usually set with the path of a bundled `index.css` (output build from `src/editor.scss`).
+- The `render` property, usually set with the path of a bundled `render.php` (output copied from `src/render.php`).
+- The `viewScript` property, usually set with the path of a bundled `view.js` (output copied from `src/view.php`).
+
+[![Open Build Output Diagram in excalidraw](https://developer.wordpress.org/files/2023/11/file-structure-build-output.png)](https://excalidraw.com/#json=c22LROgcG4JkD-7SkuE-N,rQW_ViJBq0Yk3qhCgqD6zQ "Open Build Output Diagram in excalidraw")
+
+### `index.js`
+
+The `index.js` file (or any other file defined in the `editorScript` property of `block.json`) is the entry point file for javascript files and it usually registers the block in the client. In a standard structure it imports the `edit.js` and `save.js` files to get functions required in block registration.
+
+### `edit.js`
+
+The `edit.js` file usually returns the `edit` function used in block registration that defines the React component that will be loaded on the Editor when the block is loaded or inserted.
+
+### `save.js`
+
+The `save.js` file usually returns the `save` function used in block registration that defines the markup used to save the block in the DB.
+
+### `style.scss`
+
+The `style.scss` file contains the styles of the block that will be loaded in both the editor and the frontend. In the build process this file is converted into `style-index.css` which is usually defined at `style` property in `block.json`
```

Comment:
I think it is a little missleading to only talk about SCSS here. WP scripts also supports regular straight up CSS and postCSS here.

Reply:
Thanks for this feedback 👌
I have expanded the explanation to include more CSS formats accepted by `wp-scripts`. 
Let me know what you think